### PR TITLE
DOCS-#3097: Add section regarding installation from conda-forge

### LINKED ISSUE #3097: Specify Differentiation Between Different Modin Installs
On Modin documentation, for all different installs (conda, pip, conda modin[ray], conda-forge, etc.(r, ensure user knows what backends/dependencies are supporting. i.e. For AI Kit Modin, ensure user knows OmniSci backend and Ray are uincluded, but not Dask. This way, users are aware of what underlying technical stakc is in their specific installation of Modin, and how that may differ to other install method. Best in a spreadsheet chart form.....

@aregm

### PR DESCRIPTION
**Pull request by Garra1980**
Signed-off-by: izamyati <igor.zamyatin@intel.com>

## What do these changes do?

- [x] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contributing.html)
- [x] passes `flake8 modin`
- [x] passes `black --check modin`
- [x] signed commit with `git commit -s` 
- [x] Resolves #3097
- [ ] tests added and passing
- [ ] module layout described at `docs/developer/architecture.rst` is up-to-date

### CODE REVIEW

**Comment by Garra1980** [comment]
cc @vnlitvinov

File: docs/installation.rst [hunk 1]
```diff
@@ -53,6 +53,44 @@ or for different functionalities of Modin. Here is a list of dependency sets for
...
+| modin                       | Dask                          |
++-----------------------------+-------------------------------+
+| modin-all                   | Dask, Ray, OmniSci_           |
++-----------------------------+-------------------------------+
+| modin-dask                  | Dask                          |
++-----------------------------+-------------------------------+
+| modin-ray                   | Ray                           |
++-----------------------------+-------------------------------+
+| modin-omnisci               | OmniSci                       |
++-----------------------------+-------------------------------+
```
Comment (YarShev) [inline docs/installation.rst:74]:
```suggestion
+---------------------------------+-----------------------------------+
| **Package name in conda-forge** | **Engine(s)**                     |
+---------------------------------+-----------------------------------+
| modin                           | Dask                              |
+---------------------------------+-----------------------------------+
| modin-all                       | Dask, Ray, OmniSci_               |
+---------------------------------+-----------------------------------+
| modin-dask                      | Dask                              |
+---------------------------------+-----------------------------------+
| modin-ray                       | Ray                               |
+---------------------------------+-----------------------------------+
| modin-omnisci                   | OmniSci                           |
+---------------------------------+-----------------------------------+
```

Just to highlight first line of the table.

Comment (YarShev) [reply docs/installation.rst:74]: Also, I would change the following sentence at top of the file:
```
There are a couple of ways to install Modin. Most users will want to install with pip, but some users may want to build from the master branch on the GitHub repo. The master branch has the most recent patches, but may be less stable than a release installed from pip.
```
to
```
There are a couple of ways to install Modin. Most users will want to install with pip or conda, but some users may want to build from the master branch on the GitHub repo. The master branch has the most recent patches, but may be less stable than a release installed from pip or conda.
```
(added "or conda" two times)

Comment (Garra1980) [reply docs/installation.rst:74]: Will do, thanks!

**Review by YarShev — DISMISSED** [review]
LGTM when tests pass

**Comment by Garra1980** [comment]
any comments @vnlitvinov?

File: docs/installation.rst [hunk 2]
```diff
@@ -53,6 +53,44 @@ or for different functionalities of Modin. Here is a list of dependency sets for
 
   pip install "modin[dask]" # If you want to use the Dask backend
 
+Installing with conda
+---------------------
+
+Modin releases can be installed using ``Miniconda`` from conda-forge channel. Starting from 0.10.1
```
Comment (vnlitvinov) [inline docs/installation.rst:59]:
```suggestion
Modin releases can be installed using ``conda`` from conda-forge channel. Starting from 0.10.1
```
I don't see a point of referencing `miniconda` specifically, that's just a distribution while we're speaking about the tool.

File: docs/installation.rst [hunk 3]
```diff
@@ -53,6 +53,44 @@ or for different functionalities of Modin. Here is a list of dependency sets for
...
+it is possible to install modin with chosen engine(s). Current options are:
```
Comment (vnlitvinov) [inline docs/installation.rst:60]:
```suggestion
it is possible to install modin with chosen engine(s) alongside. Current options are:
```

File: docs/installation.rst [hunk 4]
```diff
@@ -53,6 +53,44 @@ or for different functionalities of Modin. Here is a list of dependency sets for
...
++---------------------------------+---------------------------+
+| modin                           | Dask                      |
++---------------------------------+---------------------------+
+| modin-all                       | Dask, Ray, OmniSci_       |
++---------------------------------+---------------------------+
+| modin-dask                      | Dask                      |
++---------------------------------+---------------------------+
+| modin-ray                       | Ray                       |
++---------------------------------+---------------------------+
+| modin-omnisci                   | OmniSci                   |
```
Comment (vnlitvinov) [inline docs/installation.rst:73]:
```suggestion
| modin-omnisci                   | Ray, OmniSci_              |
```
to make it a link as well

also maybe other engines should have a link?..

Comment (Garra1980) [reply docs/installation.rst:73]: I made such link in modin-all line

Comment (Garra1980) [reply docs/installation.rst:73]: Added link for other engines

**Review by vnlitvinov — APPROVED** [review]

**Review by YarShev — APPROVED** [review]

**Pull request merged**